src/components/bottomSheet/BottomSheet.tsx

```diff
@@ -693,9 +694,10 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
         /**
          * store next position
          */
-        animatedNextPosition.value = position;
-        animatedNextPositionIndex.value =
-          animatedSnapPoints.value.indexOf(position);
+        const keyboardOffset = animatedKeyboardState.value == KEYBOARD_STATE.SHOWN ? animatedKeyboardHeight.value : 0;
+
+        animatedNextPosition.value = position + keyboardOffset;
+        animatedNextPositionIndex.value = position == 0 ? INITIAL_SNAP_POINT : animatedSnapPoints.value.indexOf(position + keyboardOffset);
```

Comment:
When the position was `0` (from a fully extended sheet), the index was still specified as `-1`. But that had the implication that the sheet was closed/closing.
I thought to instead set it to `INITIAL_SNAP_POINT` (-999), as that conveys a better meaning, especially in the context of the `onAnimate` callback